tools/download_utils/requests_wrappers.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper functions for easy use of requests library.
+"""
+
+import requests
+import json
+import time
+
+
+def request_url_json(url: str) -> dict:
+    """Get JSON object version of reponse to GET request to given URL.
+
+  Args:
+    url: URL to make the GET request.
+
+  Returns:
+    JSON decoded response from the GET call.
+      Empty dict is returned in case the call fails.
+  """
+    print(url)
+    try:
+        req = requests.get(url)
+        # print(req.url)
+    except requests.exceptions.ReadTimeout:
+        print('Timeout occoured, retrying after 10s.')
```

Comment:
Can add retry count and retry interval as arguments and recursively call the same function to
allow multiple retries.
```
def request_url_json(url: str, max_retries: int = 1, retry_interval: int = 10) -> dict:
...

  except requests.exceptions.ReadTimeout:
      if max_retries> 0:
           time.sleep(retry_interval)
          return request_url_json(url, max_retries - 1, retry_interval)

Reply:
fixed. Thanks for the snippet.